internal/pkg/docker/dockerengine/dockerengine.go

```diff
@@ -119,6 +121,14 @@ func (c CmdClient) Build(in *BuildArguments) error {
 		args = append(args, "--build-arg", fmt.Sprintf("%s=%s", k, in.Args[k]))
 	}
 
+	if in.ContainerName != "" {
+		in.Labels["container"] = in.ContainerName
+	}
+	// Add Labels to docker build call.
+	for k, v := range in.Labels {
+		args = append(args, "--label", fmt.Sprintf("%s=%s", k, v))
```

Comment:
Do you think we need to show the user the `docker build --label` command that is being run, like we do if we are using the `--platform` flag? I don't have a preference, just a thought.

Reply:
we do show labels to user in the `docker build command` . 
Here is what command line looks like
`Building your container image: docker build -t 1234679.dkr.ecr.us-west-2.amazonaws.com/vote/results -t 12345679.dkr.ecr.us-west-2.amazonaws.com/vote/results  --platform linux/x86_64 /Users/kolladit/demo/backend -f /Users/kolladit/demo/backend/Dockerfile`

Reply:
we do show labels to user in the docker build command .

Here is what command line looks like
`Building your container image: docker build -t 1234679.dkr.ecr.us-west-2.amazonaws.com/vote/results:nginx-latest  --platform linux/x86_64 --label builder=copilot-cli --label version=v1.26.0 --label container=nginx-latest /Users/kolladit/demo/backend -f /Users/kolladit/demo/backend/Dockerfile`

Reply:
Isn't that only `if in.Platform != ""`? ((New) line 134 below)

Reply:
yes that's right.

Reply:
I don't have a strong opinion. Do you think it would be useful for the user?